tests/sqlabstraction/query_subselect_test.php

```diff
@@ -92,7 +92,7 @@ protected function setUp()
         $this->e = $this->q->expr;
     }
 
-    public function testSubSelect()
+    public function __construct()
```

Comment:
This change looks wrong. The "testSubSelect" method isn't the constructor of this class, it's for a different class higher up in the file. I think this is a checker error. I'm OK with changing the name here, but it certainly should not be ``__construct``.

Reply:
You are right ` 7.0 | /tests/sqlabstraction/query_subselect_test.php:95                                                  | method_name      | Method name testSubSelect:TestSubSelect (@php4_constructors) is deprecated. ` seems to be a false-positive by `phpcf`. I removed that change and the test runs through again.